stbt_rig.py

```diff
@@ -1,89 +1,197 @@
 #!/usr/bin/env python2
 
+"""Command-line tool for interacting with the Stb-tester Portal's REST API.
+
+Copyright 2017 Stb-tester.com Ltd. <support@stb-tester.com>
+Released under the MIT license.
+"""
+
 import argparse
+import ConfigParser
 import logging
 import os
 import shutil
 import subprocess
 import sys
 import tempfile
 import time
-from ConfigParser import SafeConfigParser
 from contextlib import contextmanager
+from textwrap import dedent
 
+# Third-party libraries. Keep this list to a minimum to ease deployment.
 import requests
 
 
 logger = logging.getLogger("stbt_rig")
 
 
 def main(argv):
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--portal-url")
+    parser = argparse.ArgumentParser(
+        description="Command-line tool for interacting with the Stb-tester "
+        "Portal's REST API.",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=dedent("""\
+            AUTHENTICATION:
+              blah blah
+              See https://stb-tester.com/manual/rest-api-v2#authentication
+
+            INTERACTIVE MODE:
+              blah blah
```

Comment:
Hoping this documentation will be improved in future commits ;)

Hmm, maybe not?